template/api/page/models/page.settings.json

```diff
@@ -72,6 +67,14 @@
           "localized": true
         }
       }
+    },
+    "slug": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
```

Comment:
I think we should add a regex for validation, because strings are just too easy to break, with characters like spaces for example

Also, my understanding of our conversation with Alex was that we'd stop localizing the slug entirely, to only rely on the locale prefix instead